src/main/java/bio/terra/workspace/service/workspace/flight/CreateGoogleContextRBSFlight.java

```diff
@@ -7,13 +7,19 @@
 import bio.terra.workspace.common.utils.FlightBeanBag;
 import bio.terra.workspace.service.crl.CrlService;
 
-// NOTE: DO NOT USE. Currently just a shell class to exercise connection to Buffer Service.
+/**
+ * A {@link Flight} for creating a Google cloud context for a workspace using RBS (Buffer Service)
+ * to create the project.
+ */
 public class CreateGoogleContextRBSFlight extends Flight {
```

Comment:
nit: I would've expected only the first letter of a borrowed acronym to be capitalized, i.e. `CreateGoogleContextRbsFlight`.